### PR TITLE
feat: add non_ocds_scrapy project

### PR DESCRIPTION
**Pull request by yolile**

### CODE REVIEW

File: non_ocds_collect/non_ocds_scrapy/exporters.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+class HeadlessCsvItemExporter(CsvItemExporter):
+
+    def __init__(self, *args, **kwargs):
+
+        # args[0] is (opened) file handler
+        # if file is not empty then skip headers
+        if args[0].tell() > 0:
+            kwargs['include_headers_line'] = False
+
+        super(HeadlessCsvItemExporter, self).__init__(*args, **kwargs)
```
Comment (jpmckinney) [inline non_ocds_collect/non_ocds_scrapy/exporters.py]: This is old style. You can just do `super().`

Comment (jpmckinney) [inline non_ocds_collect/non_ocds_scrapy/exporters.py]:
```suggestion
    def __init__(self, file, *args, **kwargs):
        # Include the header line if the current stream position is the start of the file.
        kwargs['include_headers_line'] = file.tell() == 0
        super().__init__(file, *args, **kwargs)
```

File: non_ocds_collect/non_ocds_scrapy/filters.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+class UzbekistanAuctionFilter:
+    def __init__(self, feed_options):
+        self.feed_options = feed_options
+
+    def accepts(self, item):
+        if "product_name" in item:
+            return False
+        return True
```
Comment (jpmckinney) [inline non_ocds_collect/non_ocds_scrapy/filters.py]:
```suggestion
        return "product_name" not in item
```

File: non_ocds_collect/non_ocds_scrapy/filters.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+
+class UzbekistanAuctionProductFilter:
+    def __init__(self, feed_options):
+        self.feed_options = feed_options
+
+    def accepts(self, item):
+        if "product_name" in item:
+            return True
+        return False
```
Comment (jpmckinney) [inline non_ocds_collect/non_ocds_scrapy/filters.py]:
```suggestion
        return "product_name" in item
```

File: non_ocds_collect/requirements_dev.in [hunk 4]
```diff
@@ -0,0 +1,10 @@
+-r requirements.txt
+coverage[toml]
+coveralls
+flake8
+isort
+pip-tools
+psycopg2-binary
+pytest
+pytest-cov
+pytest-order
```
Comment (jpmckinney) [inline non_ocds_collect/requirements_dev.in]: Do we use it?

```suggestion
```

File: non_ocds_collect/requirements.in [hunk 5]
```diff
@@ -0,0 +1,7 @@
+psycopg2
+requests
+scrapy
+scrapyd
+scrapyd-client
+sentry-sdk
+twisted
```
Comment (jpmckinney) [inline non_ocds_collect/requirements.in]: Not imported directly.

```suggestion
```

File: non_ocds_collect/requirements.in [hunk 6]
```diff
@@ -0,0 +1,7 @@
+psycopg2
+requests
+scrapy
+scrapyd
+scrapyd-client
+sentry-sdk
```
Comment (jpmckinney) [inline non_ocds_collect/requirements.in]: Sentry is not configured

File: non_ocds_collect/requirements.in [hunk 7]
```diff
@@ -0,0 +1,7 @@
+psycopg2
+requests
+scrapy
+scrapyd
+scrapyd-client
```
Comment (jpmckinney) [inline non_ocds_collect/requirements.in]: Not sure if we are deploying Scrapyd, or just manually running scrapy?

File: non_ocds_collect/requirements.in [hunk 8]
```diff
@@ -0,0 +1,7 @@
+psycopg2
+requests
```
Comment (jpmckinney) [inline non_ocds_collect/requirements.in]: Not used

```suggestion
```

**Review by jpmckinney — COMMENTED** [review]
We should add https://ocp-software-handbook.readthedocs.io/en/latest/python/linting.html#pre-commit-hooks, which runs Black, etc.

I didn't read super carefully, more just to follow handbook.

File: non_ocds_collect/requirements.in (hunk 7 above)
Comment (yolile) [reply non_ocds_collect/requirements.in]: You are right, we don't need Scrapyd

**Pull request merged**